### ISSUE TITLE
CI does not upload binary

### ISSUE THREAD

**Issue by spazziale** [MollySocket/mollysocket#14]
Since version 0.1.0, binary packages are missing.

**Comment by genofire** [comment]
there was no ci for building binary ... (this was uploaded manuelle).

but the idea is good ... i am really bad in github actions (i prefer gitlab), so do you like to create another workflow for that tasks?

**Comment by p1gp1g** [comment]
Indeed, I forgot I did it manually. We can use something like this: https://github.com/UnifiedPush/fcm-distributor/blob/main/.github/workflows/main.yml#L27-L33
